src/components/irregularity/helpers/Fail.tsx

```diff
@@ -63,37 +31,37 @@ export default function Fail({ setFail, setActiveStep }: Props) {
 
   return (
     <ThemeProvider theme={theme}>
-      <Grid container spacing={7} className={classes.container}>
-        <Grid item xs={12} className={classes.gridIcon}>
-          <ErrorOutlineOutlinedIcon sx={{ transform: 'scale(5.0)', color: '#F44336' }} />
-        </Grid>
-        <Grid item xs={12} className={classes.gridItem}>
+      <Root container>
+        <Icon item xs={12} color="#F44336">
+          <ErrorOutlineOutlinedIcon />
+        </Icon>
+        <Grid item xs={12}>
           <Typography variant="h5">{t('steps.fail.title')}</Typography>
         </Grid>
-        <Grid item xs={12} className={classes.gridItem}>
+        <Grid item xs={12} my={5}>
           <Typography variant="h6">{t('steps.fail.subtitle')}</Typography>
         </Grid>
-        <Grid container item className={classes.containerButtons}>
+        <ButtonsWrapper container item>
           <Grid item>
             <Typography variant="body1">{t('steps.fail.label-campaigns')}</Typography>
-            <LinkButton href={`/campaigns`} variant="outlined" className={classes.button}>
+            <ActionLinkButton href={`/campaigns`} variant="outlined">
```

Comment:
Remove the hardcoded string
```suggestion
            <ActionLinkButton href={routes.campaigns.index} variant="outlined">
```